bin/sender.py

```diff
@@ -130,16 +184,19 @@ def main():
             path_type = 'dirq'
 
         sender = Ssm2(brokers, 
-                   cp.get('messaging','path'),
-                   path_type=path_type,
-                   cert=cp.get('certificates','certificate'),
-                   key=cp.get('certificates','key'),
-                   dest=cp.get('messaging','destination'),
-                   use_ssl=cp.getboolean('broker','use_ssl'),
-                   capath=cp.get('certificates', 'capath'),
-                   enc_cert=server_cert,
-                   verify_enc_cert=verify_server_cert)
-        
+                      cp.get('messaging', 'path'),
+                      path_type=path_type,
+                      cert=cp.get('certificates', 'certificate'),
+                      key=cp.get('certificates', 'key'),
+                      dest=cp.get('messaging', 'destination'),
+                      use_ssl=use_ssl,
```

Comment:
if using `Ssm2.AMS_MESSAGING`, `use_ssl` doesn't get set so this using it here causes an error.

Reply:
My bad. I've re-added setting it to `None`, but only when using AMS this time, for both sender and receiver.